### DISCUSSION TITLE
Dracula UI-like thingy for Catppuccin

### DISCUSSION THREAD

**Discussion by tizu69** [catppuccin/catppuccin#2236] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### Any additional comments?

possibly not just for react but also base html (the styles at least) so that this can get ported to other langs

### Submission Guidelines

- [ ] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Hey, we already have [catppuccin/ui](https://github.com/catppuccin/ui) which aims to implement this. The contributor who was interested in pushing that work forward is no longer interested so it's gone stale.

I believe @unseen-ninja and I have spoken about this but essentially we'd want to think for a little bit about Catppuccin's brand presence and design language before continuing the implementation for that.
